Product/product.html

```diff
@@ -0,0 +1,233 @@
+                <span class="icon-badge flex-align-center">14</span>
+                </div></a>
+            </li>
+            <li><a href="/Wishlist/wishlist.html"><i class="bi bi-heart"></i></a></li>
+            <li><a href="#"><i class="bi bi-person-fill"></i></a></li>
+        </ul>
+    </nav> 
+    <div class="flex gap-2">
+        <div class="filters"> 
+            <div class="flex pd-1">
+                <span>Filter</span>
+                <span class="text-accent ml-5">Clear All</span>
+            </div>
+            <hr>
+            <div class="pd-1">
+                <p class="text-dark">Price</p>
+                <input type="range" min="1" max="10000" value="50" id="priceRange">
+                <p id="price" class="text-gray text-xsm"></p>
+            </div>
+            <hr>
+            <div class="mb-1 mt-1">
+                <ul class="list-item">
+                    <li>Categories</li>
+                    <li><input type="checkbox" ></input><span>Jackets</span></li>
+                    <li><input type="checkbox" ></input><span>Shoes</span></li>
+                    <li><input type="checkbox" ></input><span>Trolly</span></li>
+                    <li><input type="checkbox" ></input><span>Bags</span></li>
+                    <li><input type="checkbox" ></input><span>Shirts</span></li>
+                </ul>
+                </div>
+            <hr>
+            <div class="mt-1 mb-1">    
+                <ul class="list-item">
+                    <li>Rating</li>
+                    <li><input type="radio" name="rating" id="5-star" ></input><span>5 star</span></li>
+                    <li><input type="radio" name="rating" id="4-star" ></input><span>4 star</span></li>
+                    <li><input type="radio" name="rating" id="3-star" ></input><span>3 star</span></li>
+                    <li><input type="radio" name="rating" id="2-star" ></input><span>2 star</span></li>
+                    <li><input type="radio" name="rating" id="1-star" ></input><span>1 star</span></li>
+                </ul>
+            </div>
+            <hr>
+            <div class="mt-1">    
+                <ul class="list-item">
+                    <li>Sort By</li>
+                    <li><input type="radio" name="rating" id="high-to-low" ></input><span>Price - High to Low</span></li>
+                    <li><input type="radio" name="rating" id="low-to-high" ></input><span>Price - Low to high</span></li>
+                </ul>
+            </div>
+        </div>
+        <div class="flex-col">
+        <div class="grid-three-items product-container">
+            <div class="grid-item-1 w-200 ">
+                <div class="card box-shadow-bottom">
+                    <img src="/assest/jacket.jpg" class="img-md"/>
+                    <div class="card-body">
+                        <div class="flex justify-space-between">
+                            <div class="card-title ">Jacket</div>
```

Comment:
suggestion: instead of just using div, use any semantic html tag, you could have written p/h html tag.
appreciation: UI is looking great. Cards are spacious. Great work.